deploy/kubernetes/csi-nfs-node.yaml

```diff
@@ -14,6 +14,8 @@ spec:
       labels:
         app: csi-nfs-node
     spec:
+      hostNetwork: true
```

Comment:
Why is host network needed? I had explicitly removed hostNetwork so that you could give an incluster hostname as the nfs server.

Reply:
`hostNetwork: true` is a must, I have tried after node driver daemonset  restarted, if there is no `hostNetwork: true` setting, original nfs connection would be broken. Same issue as https://github.com/kubernetes-csi/csi-driver-smb/issues/130

Reply:
@msau42 PTAL, thanks.

Reply:
thanks for the explanation, this makes sense. Can you add a comment here?

Reply:
added a comments in yaml config, PTAL.